fancoil/chiltrix-htp-fancoil-documentation.md

```diff
@@ -42,17 +42,19 @@ Vesion	1.0			update:
 | 28301   | 6E8D | 03/10         | On/Off                                   | 0=off,1=on                                                                                                            | DIGI1  |
 | 28302   | 6E8E | 03/10         | Mode                                     | 0～auto；1～cooling;2～dehumidification；3～ventilate；4～heating；                                                            | DIGI1  |
 | 28303   | 6E8F | 03/10         | Fanspeed                                 | 2～low speed；3～medium speed； 4～high speed；<br>6～aotu                                                                   | DIGI1  |
-| 28306   | 6E92 | 03/10         | Timer off                                |                                                                                                                       | DIGI1  |
-| 28307   | 6E93 | 03/10         | Timer off                                |                                                                                                                       | DIGI1  |
+| 28306   | 6E92 | 03/10         | Timer off                                | 1-11=set a timer in hours after which to turn unit off.                           
```

Comment:
Does the timer start when set by updating this register value?

Reply:
I made these reference a new data type with notes on what I have observed from my testing so far. I suspect there's a way to reset these timers but it's very tedious to explore because you need to wait an hour for every experiment!